lib/looker-sdk/authentication.rb

```diff
@@ -51,7 +51,8 @@ def authenticate
 
       set_access_token_from_params(nil)
       without_authentication do
-        post("#{URI.parse(api_endpoint).path}/login", {}, :query => application_credentials)
+        encoded_auth = Faraday::Utils.build_query application_credentials
```

Comment:
parens around the param would have been nice :)

Reply:
I gave you a PR https://github.com/looker/looker-sdk-ruby/pull/94

Accept it now so it will be nicer the next time we do a release.